base/pkg/entry.jl

```diff
@@ -553,6 +553,20 @@ function resolve(
         end
         rethrow(err)
     end
+    imported = String[]
+    for (pkg,(ver1,ver2)) in changed
+        if ver1 !== nothing && ver2 !== nothing
```

Comment:
I've found most of these internal Pkg functions really confusing and non-obvious in terms of what they're supposed to return and when, every time I've looked at them.